src/udev/udevd.c

```diff
@@ -1509,6 +1509,14 @@ static int on_inotify(sd_event_source *s, int fd, uint32_t revents, void *userda
                 _cleanup_(sd_device_unrefp) sd_device *dev = NULL;
                 const char *devnode;
 
+                /* Do not handle IN_IGNORED here. The same watch handle may be already reused for
+                 * another device node. The old symlink will be removed by a worker which processes the
```

Comment:
hmm, watch handles can be reused? i wasn't aware… under which conditions?

Reply:
Not sure, but possibly. I will rephrase the comment.